assets/images/ONNX-Runtime-logo.svg

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
there's already a logo image in gh-pages. we should reference that instead of having another copy,

Reply:
Consolidated into one directory.